tests/diagramconfiguration_test.py

```diff
@@ -695,3 +696,18 @@ def test__callable_tree_item__export_project_with_std_function__export_correct_d
 @enduml'''.format(file_name, directory)
 
     assert diagram == expected_diagram
+
+
+def test__generate_uml__export_project__show_generated_uml(directory):
```

Comment:
nit-pick: suggest to split into two tests: one covering the initial situation (pixmap is `None`) and the other one covering the state after an export

Reply:
Since I already changed the way of the uml generation to svg, I don't think it makes a lot of sense to spend changing the tests on this version. But I'll split them in the newer tests.